<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GwConnectionDialog</class>
 <widget class="QDialog" name="GwConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Class</string>
       </property>
       <property name="buddy">
        <cstring>classComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classComboBox">
       <item>
        <property name="text">
         <string>Choose Class</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JSS 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JSS 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JSS 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SS 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SS 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SS 3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Stream</string>
       </property>
       <property name="buddy">
        <cstring>classStreamComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classStreamComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Term</string>
       </property>
       <property name="buddy">
        <cstring>termComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="termComboBox">
       <item>
        <property name="text">
         <string>Select Term</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1st</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2nd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3rd</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Academuc Session</string>
       </property>
       <property name="buddy">
        <cstring>sessionLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sessionLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>classComboBox</tabstop>
  <tabstop>classStreamComboBox</tabstop>
  <tabstop>termComboBox</tabstop>
  <tabstop>sessionLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GwConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GwConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
